peps/pep-0723.rst

```diff
@@ -101,14 +102,38 @@ This PEP defines a metadata comment block format loosely inspired [2]_ by
 
 __ https://docutils.sourceforge.io/docs/ref/rst/directives.html
 
-Any Python script may have top-level comment blocks that start with the line
-``# /// TYPE`` where ``TYPE`` determines how to process the content, and ends
```

Comment:
Is TYPE allowed to be arbitrary characters? If not, the valid form of TYPE should be specified. The regex allows letters, numbers and dashes. So according to both the regex and the text, the following is valid

```
# /// ---------
```

(with a type of `---------`). According to the text,

```
# /// *&^%£€#
```

is valid (with a type of `*&^%£€#`) but the regex disallows this.